db/migrate/20220219192642_add_post_ref_to_comment.rb

```diff
@@ -0,0 +1,7 @@
+class AddPostRefToComment < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :comments, :post, null: false, foreign_key: true
+    add_column :comments, :PostId, :integer
+    add_index :comments, :PostId
```

Comment:
- [ ] Kindly remove `add_column` and `add_index`. The column and the index are automatically created with `add_reference`
- [ ] make the above change for `likes`, and `posts` as well.